doc/src/manual/embedding.md

```diff
@@ -241,6 +241,17 @@ jl_value_t *jl_call(jl_function_t *f, jl_value_t **args, int32_t nargs)
 Its second argument `args` is an array of `jl_value_t*` arguments and `nargs` is the number of
 arguments.
 
+There is also an alternative, possibly simpler, way of calling Julia functions and that is via [`@cfunction`](@ref).
+Using `@cfunction` allows you to do the type conversions on the Julia side which typically is easier than doing it on
+the C side. The `sqrt` example above would with `@cfunction` be written as:
+
+```c
+double (*sqrt_ptr)(double) = jl_unbox_voidpointer(jl_eval_string("@cfunction(sqrt, Float64, (Float64,))"));
+double ret = (*sqrt_ptr)(2.0);
```

Comment:
You can just call it as `sqrt_ptr(2.0)`, I think?

Reply:
For that reason, I think it's clearer to just name it like you would an ordinary C function, e.g. call it `sqrt_jl` instead of `sqrt_ptr`.

Reply:
I see that @vtjnash made the same point below.

Reply:
Fixed